stable/README.md

```diff
@@ -86,19 +83,30 @@ Each Helm Chart has a default values.yaml parameter file that contains configura
 
 ## Deployment Steps
 
-**Note:** You MUST first disable Linux Transparent Huge Pages(THP) on all cluster nodes that will host NuoDB pods. Run the `transparent-hugepage` chart first.
+1. Disable Linux Transparent Huge Pages(THP) on all cluster nodes that will host NuoDB pods.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
I guess these whitespaces are added by the editor.

Reply:
all of these should be removed

Reply:
They were not added by the editor. I added them for indentation to format the text better.